nevergrad/functions/functionlib.py

```diff
@@ -63,6 +65,43 @@ def _short_repr(self) -> str:
         return "Photonics"
 
 
+class ML_tuning(ExperimentFunction):
+    """Class for generating ML hyperparameter tuning problems.
+    """
+    def __init__(self, problem_type: str):
+        self.name = problem_type
+        self._parameters = {x: y for x, y in locals().items() if x not in ["__class__", "self"]}
+        self.rng = np.random.RandomState(1)
+
+        def decision_tree_parametrization(depth: int):
```

Comment:
please never use inline functions, add this as a private function of the class

Reply:
👍